npm/ng-packs/scripts/build.ts

```diff
@@ -32,6 +32,7 @@ import program from 'commander';
         'copy',
         '--angular',
         '--no-watch',
+        '--sync',
```

Comment:
`--sync` will slow down the build process.

Reply:
I know, but Angular9  compiler is not happy with parallel builds. It throws an exception stating that there is another compiler running. check this out: https://github.com/angular/angular/issues/32431